9.2/root/usr/bin/hooks.d/post-init

```diff
@@ -0,0 +1 @@
+#!/bin/bash
```

Comment:
I was not sure what to show in example file. I really like the idea what you are proposing @hhorak . Where would we put these examples?
+One more specific question. Which SQL commands we run during initialization? I have been thinking about creating user, database and table. Is this enough or you think about something more complex?